File: ugc/src/main/java/com/inappstory/sdk/ugc/picker/FilePreviewsHolder.java
package com.inappstory.sdk.ugc.picker;

import android.view.View;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

class FilePreviewsHolder extends RecyclerView.ViewHolder {

    boolean isActive = false;
    String path;

    public FilePreviewsHolder(@NonNull View itemView) {
        super(itemView);
    }
}

File: ugc/src/main/java/com/inappstory/sdk/ugc/picker/FilePreviewsHolder.kt
package com.inappstory.sdk.ugc.picker

import android.view.View
import androidx.recyclerview.widget.RecyclerView

internal class FilePreviewsHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var isActive = false
    @JvmField
    var path: String? = null
}